src/components/ParticipantInfo/ParticipantInfo.test.tsx

```diff
@@ -116,7 +103,6 @@ describe('the ParticipantInfo component', () => {
   //     <ParticipantInfo
   //       onClick={() => {}}
   //       isSelected={false}
-  //       gridView={false}
   //       participant={{ identity: 'mockIdentity' } as any}
   //     >
   //       mock children
```

Comment:
Do we want all the commented tests?
Maybe we have them but as a different variation, if so - I'd suggest deleting it to clean up the code

Reply:
its test that not work, for now, we need to fix them, there is a story on that